### ISSUE TITLE
Unknown printer error

### ISSUE THREAD

**Issue by emanuelen5** [makerspace/memberbooth#80]
# Code version
b5e216a

# Stack trace:
```python
2020-10-05 21:15:25,561 ERROR [/home/memberbooth/memberbooth/src/gui/states.py:78]: This error should not occur
Traceback (most recent call last):
  File "/home/memberbooth/memberbooth/lib/python3.7/site-packages/PIL/Image.py", line 2656, in open
    fp.seek(0)
AttributeError: 'Label' object has no attribute 'seek'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/memberbooth/memberbooth/lib/python3.7/site-packages/brother_ql/conversion.py", line 84, in convert
    im = Image.open(image)
  File "/home/memberbooth/memberbooth/lib/python3.7/site-packages/PIL/Image.py", line 2658, in open
    fp = io.BytesIO(fp.read())
AttributeError: 'Label' object has no attribute 'read'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/memberbooth/memberbooth/src/gui/states.py", line 60, in gui_print
    print_status = label_printer.print_label(label)
  File "/home/memberbooth/memberbooth/src/label/printer.py", line 16, in print_label
    qlr = convert(qlr, [label], LABEL_TYPE)
  File "/home/memberbooth/memberbooth/lib/python3.7/site-packages/brother_ql/conversion.py", line 86, in convert
    raise NotImplementedError("The image argument needs to be an Image() instance, the filename to an image, or a file handle.")
NotImplementedError: The image argument needs to be an Image() instance, the filename to an image, or a file handle.
2020-10-05 21:15:25,565 ERROR [/home/memberbooth/memberbooth/src/gui/design.py:27]: GUI error: Unknow printer error occured!
```

**Comment by InMyOrbit** [comment]
Solved in commit 56bfa79e975b813b7ce37eeedc140c0c739231f6.

**Comment by emanuelen5** [comment]
Tested with printer and seems to work!
